File: tests/java/LongIdentifierTest/LongIdentifierTest2.java
// The class has method that calls to a field with a name longer than 30.
// -> Has long identifier smell.
public class LongIdentifierTest2 //Type originates from: LongIdentifierTest2.h
{
	private int thisIsAVeryVeryVeryLongFieldName;
	private void fun1(int p) //Method definition originates from: LongIdentifierTest2.cpp
	{
		int v = this.thisIsAVeryVeryVeryLongFieldName;
	}
}

File: tests/cpp/LongIdentifierTest/LongIdentifierTest2.cpp
// The class has method that calls to a field with a name longer than 30.
// -> Has long identifier smell.
#include "LongIdentifierTest2.h"

void LongIdentifierTest2 :: fun1(int p) {
    int v = this->thisIsAVeryVeryVeryLongFieldName;
}